README.md

```diff
@@ -662,6 +675,53 @@ which help you to use the various OpenAPI 3 Authentication mechanism.
     type ObjectCategory int
     ```
 
+- `x-oapi-codegen-middlewares`: specifies a list of middlewares. These can be middlewares that are
```

Comment:
What is conventiof for naming these? Maybe a simpler name, such as `x-middlewares`, is enough?